plantcv/__init__.py

```diff
@@ -8,7 +8,7 @@
            'crop_position_mask', 'get_nir', 'adaptive_threshold', 'otsu_auto_threshold', 'report_size_marker_area',
            'white_balance', 'triangle_auto_threshold', 'acute_vertex', 'scale_features', 'landmark_reference_pt_dist',
            'x_axis_pseudolandmarks', 'y_axis_pseudolandmarks', 'gaussian_blur', 'cluster_contours',
-           'cluster_contour_splitimg', 'rotate_img', 'shift_img', 'output_mask', 'auto_crop',
+           'cluster_contour_splitimg', 'rotate_img', 'rotate_img2','shift_img', 'output_mask', 'auto_crop',
```

Comment:
I think these should be the function names, not the file name, so `rotate` instead of `rotate_img2`.